static/js/app.ts

```diff
@@ -874,15 +874,26 @@ function getConfettiForAdventure(adventure: string){
 
   switch (adventure) {
     case 'turtle':
-      return [['🐢']];
+      let turtleEmoji = Array.from(ErrorMessages['Emoji_turtle'])
```

Comment:
With the dict in yaml, we can do something like `ErrorMessages[adventure]` here, reducing the neem for the switch.

Reply:
Great suggestion! I've pushed a new commit.